build_test/test/resolve_source_test.dart

```diff
@@ -12,28 +12,22 @@ import 'package:test/test.dart';
 void main() {
   group('should resolveSource of', () {
     test('a simple dart file', () async {
-      var resolverDone = new Completer<Null>();
-      var resolver = await resolveSource(r'''
+      var libExample = await resolveSource(r'''
```

Comment:
nit: maybe in one of these examples put the expect inside the `action` callback? It would be good just to confirm that works as we want and also as an additional example of how you can use the resolver for multiple things without having to use the `tearDown` logic.